cmake/CudaConfig.cmake

```diff
@@ -0,0 +1,113 @@
+#CUDA configuration script for AMBER
+
+# With CMake 3.7, FindCUDA.cmake crashes when crosscompiling.
+
+if(CROSSCOMPILE)
+	message(STATUS "CUDA disabled when crosscompiling.")
+	set(CUDA FALSE)
+else()
+
+	# first, find CUDA.
+	find_package(CUDA)
+	option(CUDA "Build ${PROJECT_NAME} with CUDA GPU acceleration support." FALSE)
```

Comment:
Since `CUDA` is passed from the command line `-DCUDA=TRUE`, why do we still set a boolean variable for it?

Reply:
The option is there to create a cache variable which can be set on the command line, or using the config GUI.  Basically, the option() call handles the case where you _don't_ pass `-DCUDA` on the command line, setting a false default.